util/waiter.go

```diff
@@ -20,6 +20,7 @@ type Waiter struct {
 func NewWaiter(timeout time.Duration, logInitialWait func()) *Waiter {
 	return &Waiter{
 		log:     logInitialWait,
+		updated: time.Now(),
```

Comment:
Das ist falsch- es gab ja kein Update!

Reply:
Das ist Absicht.
updated wird ausschließlich in Overdue benutzt.
Das Update am Anfang wird gebraucht, damit nach dem Start erst nach timeout Warnungen geschrieben werden.